### PR TITLE
Move attributes in the corresponding default files and disable unit tests

### PR DESCRIPTION
**Pull request by francesco-giordano**
### Description of changes
* Move attributes in the corresponding default files
* Disable unit tests. The tests can be started manually
* https://github.com/aws/aws-parallelcluster-cookbook/pull/1793

### Tests
* Manual tested

Please review the [guidelines for contributing](../CONTRIBUTING.md) and [Pull Request Instructions](https://github.com/aws/aws-parallelcluster/wiki/Git-Pull-Request-Instructions).

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: .github/workflows/dokken-validate.yml [hunk 1]
```diff
@@ -13,8 +13,6 @@ jobs:
           - ubuntu18
           - ubuntu20
           - redhat8
-        suite:
-          - aws-parallelcluster-install
```
Comment (awslitvit) [inline .github/workflows/dokken-validate.yml]: Why did you remove these lines? 
Even if we don't need them now, we can keep them for when we'll have more suites to run in parallel, can't we?

Comment (francesco-giordano) [reply .github/workflows/dokken-validate.yml]: Since now it is not mandatory but we can keep it there like a place holder: I'm going to add the lines again.

**Review by awslitvit — APPROVED** [review]
